src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/AllServicesTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests;
+
+import com.google.common.collect.ImmutableList;
+import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigList;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.model.Classification;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.AbstractMap.SimpleEntry;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.toMap;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AllServicesTest {
+
+    // value - list of services
+    private static final Map<Classification, List<Object>> servicesPerClassification = ConfigFactory
+        .load()
+        .getObject("tests.all-services")
+        .entrySet()
+        .stream()
+        .map(entry -> new SimpleEntry<>(
+            Classification.valueOf(entry.getKey()),
+            ((ConfigList) entry.getValue()).unwrapped()
+        ))
+        .collect(toMap(SimpleEntry::getKey, SimpleEntry::getValue));
+
+    @Test
+    void should_verify_all_services_are_in_the_test_config() {
```

Comment:
This test tests some internal stuff in tests themselves. It's not a functional/end-to-end test. As mentioned in the other comment, best to just define the config in this class as a map or sth.